package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "license": "UNLICENSED",
```

Comment:
MIT?

Reply:
Think there should be an name and description in here too: https://docs.npmjs.com/cli/v7/configuring-npm/package-json

Reply:
When is this nodejs packages used? I see no nodejs in `docker-compose.yml` or in actions?

Reply:
See the readme about markdown lint

Reply:
This is tooling setup, not part of the application, so I consider this to accurate
```json
  "license": "UNLICENSED",
  "private": true,
```